src/BsvUnifiedMerklePathView.tsx

```diff
@@ -0,0 +1,76 @@
+import {FC} from "react";
+import {useMerklePath} from "./MerkleProofsProvider.tsx";
+import * as _ from "lodash";
+import {NoTransactionSelected} from "./NoTransactionSelected.tsx";
+import './BsvUnifiedMerklePathView.css'
+import {MerkleProof, MerkleProofByTx, TreePart} from "./merkle-tree-data";
+
+interface BsvUnifiedMerklePathViewProps {
+
+}
+
+type BumpLeaf = {
+    hash: string,
+    txid?: true,
+    offset: number,
+} | {
+    offset: number,
+    duplicate: true
+}
+
+interface MerklePathLeaf {
+    hash: string,
+    txid?: true,
+    duplicate?: boolean,
+    offset: number,
+    height: number,
+}
+
+export const BsvUnifiedMerklePathView: FC<BsvUnifiedMerklePathViewProps> = () => {
```

Comment:
Then it can also be removed here
```suggestion
export const BsvUnifiedMerklePathView = () => {
```